jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -62,12 +55,14 @@
     "express": "^4.18.1",
     "istanbul-instrumenter-loader": "^3.0.1",
     "istanbul-lib-coverage": "^3.2.0",
+    "jsdom": "^21.0.0",
     "json-server": "^0.17.0",
     "nodemon": "^2.0.20",
     "nyc": "^15.1.0",
     "sass": "^1.54.9",
-    "sass-loader": "^13.0.2",
     "sinon": "^14.0.0",
+    "vite": "^4.0.4",
+    "vitest": "^0.27.1",
```

Comment:
Vitest is recommended in Vue3 docs https://vuejs.org/guide/scaling-up/testing.html#unit-testing